### DISCUSSION TITLE
loadouts buttons

### DISCUSSION THREAD

**Discussion by katastrophe92** [Rem0o/FanControl.Releases#806] [Ideas]
any chance to make loadouts without using the load config option?

**Comment by Rem0o** [comment]
Could you be more precise? If you have something specific in mind, a drawing or screenshot would do wonder.

**Reply by katastrophe92** [reply]
![fancontrol loadouts](https://user-images.githubusercontent.com/103950507/169535841-949cf158-6d06-4a4e-8d68-9a8456530c92.PNG)

something like this. I have saved 2 .json files: 1 is gaming profile and 1 is silent profile

**Reply by Rem0o** [reply]
Are you aware of this? :

![image](https://user-images.githubusercontent.com/14118956/169560428-a4172c86-3faf-4815-bd32-950c64fcdcd7.png)

**Reply by katastrophe92** [reply]
no lol. that makes my life easier. but still my idea is not that bad tho

**Reply by Rem0o** [reply]
Well, considering the use case, where the FanControl window won't be open since you would be doing other stuff, it would be more click/more complicated to do the same thing. Also, your idea doesn't scale well. Which config would be (1) and which would be (2) ? What about if you got 5 configs?

**Reply by katastrophe92** [reply]
it is just a example. it is enough space for more buttons and the button can be named or it can be chosen for example silent, power, extrem, gaming. anyway that helped me alot, thanks

**Reply by leftcoastfunk** [reply]
> Are you aware of this? :
> 
> ![image](https://user-images.githubusercontent.com/14118956/169560428-a4172c86-3faf-4815-bd32-950c64fcdcd7.png)

Sorry if it's annoying to post on a month-old thread, but came here looking to make a similar suggestion to what katastrophe posted

I had been using the right-click method in the taskbar like you show here, but then I noticed the next time I open the main Fan Control window, the "configuration changed to:..." message appears at the bottom _for every single time I changed it in the taskbar popup menu_!!! It was annoying enough I just started changing the profile from the main window

Additionally, sometimes I have the main window already open anyway, so in those cases it might actually be extra steps to use the right-click menu in the taskbar

just some thoughts

you've done really great with this app overall, thanks!!

**Reply by leftcoastfunk** [reply]
> Well, considering the use case, where the FanControl window won't be open since you would be doing other stuff, it would be more click/more complicated to do the same thing. Also, your idea doesn't scale well. Which config would be (1) and which would be (2) ? What about if you got 5 configs?

Alternatively, there could be clean & simple icons on the top bar allowing the user direct access to save configuration or load configuration without going to the menu first. Obviously this only helps in times the main window is open, but I always appreciate when the most frequently used controls (save, save as, and load in this case) can be removed from menus and given direct-access in the main UI...why click twice if you could click once :)
